ChatQnA/docker/gaudi/docker_compose.yaml

```diff
@@ -195,6 +195,20 @@ services:
     ipc: host
     restart: always
 
+  chaqna-gaudi-conversation-ui-server:
```

Comment:
For the "docker_compose.yaml" file, only one UI image is needed. You can mention in the README that users can modify the UI image in the "docker_compose.yaml" file and choose either "opea/chatqna-ui" or "opea/chatqna-conversation-ui".

Reply:
Agreed, Done.